paint.js

```diff
@@ -101,8 +107,21 @@ function PointerHandler(event) {
         };
         var eventType = event.type == "pointerdown" ? "touchstart" :
             event.type == "pointerup" ? "touchend" : "touchmove";
-        drawTouch(fakeTouch, eventType);
         event.preventDefault();
+        drawTouch(fakeTouch, eventType, false);
```

Comment:
Yeah looks like this isn't really behaving like a drawing app (it also wouldn't offset the Y position), just visualizing the points.  Seems fine to me.  When we ship this API we should publish a simple drawing app demo that uses it the way we'd expect people to.